views/apps/new.html

```diff
@@ -11,10 +11,7 @@ <h2 class="font-bold text-2xl font-headline mb-2">
 
   <ul class="mb-8">
     <li class="mb-2 relative pl-6">
-      <i
-        class="fas fa-check mr-3 text-green-700 absolute left-0 top-1"
-        aria-hidden="true"
-      ></i>
+      <span class="mr-3 absolute left-0">&#9989;</span>
```

Comment:
`&#9989;` is &#9989;
this should have even better compatibility than pasting an emoji.